tsconfig-base.json

```diff
@@ -21,5 +21,6 @@
     "composite": true,                     /* Ensure TypeScript can determine where to find the outputs of the referenced project to compile project. */
     "incremental": true,                   /* Enable incremental compilation by reading/writing information from prior compilations to a file on disk. */
     "useDefineForClassFields": false,      /* Use Object.defineProperty for introducing class fields. */
+    "skipLibCheck": true,                  /* Skip type checking of declaration files. */
```

Comment:
This seems unrelated, was this an error?

Reply:
Yeah this is unrelated but does not hurt :) It's a recommended setting in TypeScript and could make builds somewhat faster.